docs/modelserving/v1beta1/triton/torchscript/README.md

```diff
@@ -138,15 +139,15 @@ $ inferenceservice.serving.kserve.io/torchscript-cifar10 created
 
 
 ### Run a prediction with curl
-The first step is to [determine the ingress IP and ports](../../../../get_started/first_isvc.md#3-determine-the-ingress-ip-and-ports) and set `INGRESS_HOST` and `INGRESS_PORT`
+The first step is to [determine the ingress IP and ports](https://kserve.github.io/website/get_started/first_isvc/#3-determine-the-ingress-ip-and-ports) and set `INGRESS_HOST` and `INGRESS_PORT`
 
 The latest Triton Inference Server already switched to use KServe [prediction V2 protocol](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2), so
 the input request needs to follow the V2 schema with the specified data type, shape.
 ```bash
 MODEL_NAME=cifar10
 INPUT_PATH=@./input.json
```

Comment:
We can add a link to the input file here for user to download the input

Reply:
Added in [c98d596](https://github.com/kserve/website/pull/127/commits/c98d596da3dc525a09c5d9601415df6826b2c73b)